src/stories/MainMenu.stories.tsx

```diff
@@ -13,7 +14,10 @@ export default {
 } as ComponentMeta<typeof MainMenu>;
 
 // More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
-const Template: ComponentStory<typeof MainMenu> = (args) => <MainMenu {...args} />;
-
+const Template: ComponentStory<typeof MainMenu> = (args) => (
+	<ClientProvider>
```

Comment:
I feel like these changes are indicators this stuff should probably not be in storybook. thoughts?